packages/components/src/index.native.js

```diff
@@ -116,6 +115,8 @@ export { setClipboard, getClipboard } from './mobile/clipboard';
 export { default as AudioPlayer } from './mobile/audio-player';
 export { default as Badge } from './mobile/badge';
 export { default as Gridicons } from './mobile/gridicons';
+export { default as HR } from './mobile/hr';
+export { HorizontalRule } from './mobile/hr/horizontal-rule';
```

Comment:
This current `HorizontalRule` export doesn't appear to be consumed anywhere. Should consider it private? Possibly define both components in a single file and only export one?

```js
// primitives/src/horizontal-rule/index.native.js:
function HR () {
  // ...simple line...
}

export function HorizontalRule() {
  // ...line with optional text...
}
```

Alternatively, I wonder if it would be more appropriate to align with the web components API as much as possible, i.e. consider our "line with text" component something completely separate from a `HorizontalRule` or `HR`. This would allow the props and behavior of `HorizontalRule` to match between web and native, possibly reducing confusion. So, we would have two files: 

```js
// primitives/src/horizontal-rule/index.native.js:
export function HorizontalRule () {
  // ...simple line...
}

// components/src/mobile/labeled-horizontal-rule/index.native.js:
export function LabeledHorizontalRule() {
  // ...line with optional text...
}
```

Lastly, the component/export name of `HR` appears to be mobile-specific. Web solely relies upon `HorizontalRule`. It might make sense to align with web in regards to naming and avoid `HR`. WDYT?